### PR TITLE
[DOCS][KSPM] Adds section for Findings to the Elastic Sec UI overview

### LINKED ISSUE #2273: [DOCS][KSPM][8.4] Add info about Findings page to Elastic Security UI page
## Description 

Add a brief section to the Elastic Security UI page describing the new KSPM Findings page and linking to the Kubernetes security posture management doc that explains how to set up the KSPM integration.

## Acceptance Test Criteria

Approved by Tinsae and docs team

## Notes

Meta issue: #2260

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2273 by adding a brief section to the Elastic Security UI overview page that mentions the new Findings page, and links to more information.

Preview: [Findings section](https://security-docs_2333.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html)

Meta: #2260

### CODE REVIEW

File: docs/getting-started/security-ui.asciidoc [hunk 1]
```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
+[float]
+=== Findings page
+
+The Findings page provides an overview of your Kubernetes infrastructure configuration's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
```
Comment (joepeeples) [inline docs/getting-started/security-ui.asciidoc:97]: "Kubernetes infrastructure configuration's performance" is a long construction; could we shorten, with "configuration" implied as part of the infrastructure?
```suggestion
The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
```

File: docs/getting-started/security-ui.asciidoc [hunk 2]
```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
...
+
+[role="screenshot"]
+image::cloud-native-security/images/findings-page.png[The Findings page]
```
Comment (joepeeples) [inline docs/getting-started/security-ui.asciidoc:100]: This image looks a little fuzzy on my monitor -- usually that happens if the image was scaled down when it was saved. Snagit has an option to do this, but I always make sure it's unchecked to get a higher-quality image:
image

**Review by joepeeples — APPROVED** [review]
Looks good! A couple of minor suggestions.

**Review by nastasha-solomon — APPROVED** [review]
Nothing to add to @joepeeples 's feedback. LGTM!

File: docs/getting-started/security-ui.asciidoc [hunk 3]
```diff
@@ -91,6 +91,14 @@ The Alerts page allows you to view and manage all alerts to monitor activity wit
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
+[float]
+=== Findings page
+
+The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+
+[role="screenshot"]
+image::cloud-native-security/images/findings-page.png[The Findings page]
```
Comment (joepeeples) [inline docs/getting-started/security-ui.asciidoc:100]: Image still needs a little TLC. If it's not possible to re-do with higher res (maybe dev provided it), no worries. Just add the border so it matches the other screenshots on the UI page.

**Review by joepeeples — COMMENTED** [review]
Same comment as before about screenshot.

**Review by jmikell821 — COMMENTED** [review]
Ditto -- agree w/ previous feedback; nothing else to add on my end. We'll need to merge this as soon as we can so I can update this topic tomorrow. Thanks!

**Pull request merged**